be/src/storage/utils.cpp

```diff
@@ -119,7 +119,7 @@ Status move_to_trash(const std::filesystem::path& file_path) {
 Status copy_file(const string& src, const string& dest) {
```

Comment:
You can merge the two copy_file function in utils/file_utils.cpp and storage/utils.cpp

Reply:
done, change in https://github.com/StarRocks/starrocks/pull/4347/commits/f4f3970c240d154c4675605850fa37ac4b5a6e5e